AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLicensedImageViewLabelTest.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminAdobeStockLicensedImageViewLabelTest">
+        <annotations>
+            <features value="AdobeStockImagePanel"/>
+            <stories value="[Story #22] User views licensed images in the grid"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/309"/>
+            <title value="User views licensed label for licensed images in Adobe Stock Panel"/>
+            <description value="User views licensed label for licensed images in Adobe Stock Panel"/>
```

Comment:
Could you add a `<testCaseId>` element linking to our hiptest manual test cases, please? Like this:

```
<testCaseId value="https://app.hiptest.com/projects/131313/test-plan/folders/1051731/scenarios/3579363"/>
```

Reply:
I actually removed that since I didn't know what URL to populate for the value attribute. Do you want me to add the URL you've mentioned above?

Reply:
Yes please :)

Reply:
Done, thanks!